pdf.mod/pdf.bmx

```diff
@@ -64,7 +64,22 @@ Type TPDFDoc
 	Function GetLastError:TPDFException()
 		Return lastError
 	End Function
-
+	
+	Rem
+	bbdoc: Load TrueType font from external .ttf file and register it in the document object.
+	about:
+	<a href="../examples/load_font.bmx">Example source</a>
+	EndRem
+	Method LoadTTFontFromFile:String(path:String, embedding:Int=True)
```

Comment:
The problem with this is it doesn't support multi byte file names.

It might be better to call it `LoadTTFont` and accept an `path:Object` and have it accept a TStream or String.
And then call `LoadTTFontFromStream`, passing in a specially constructed HPDF_Stream using something like `HPDF_CallbackReader_New`, which would wrap the callback functions into the TStream.